src/tests/CodeStyleTest.js

```diff
@@ -0,0 +1,3 @@
+describe('Code style', () => {
+  it('should conform to standard', require('mocha-standard')).timeout(0)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['it' is not defined.](https://app.codacy.com/gh/tobias-inc/TobiasBot/pullRequest?prid=5027451)